scss/_patterns_code-copyable.scss

```diff
@@ -10,33 +10,45 @@ $cc-button-size: 36 * $px;
     @extend %vf-has-round-corners;
     @extend %vf-is-bordered;
     display: flex;
+    overflow: hidden;
     position: relative;
 
     & + & {
       margin-top: 0; // overrides p + p
     }
 
     &__input {
-      @include vf-icon-code($color-mid-dark);
-      background-position: $sph-intra--condensed center;
-      background-repeat: no-repeat;
       border: 0;
       color: $color-mid-dark;
       font-family: unquote($font-monospace);
       line-height: map-get($line-heights, default-text);
-      padding: 0 $cc-button-size + $sph-intra--condensed 0 $sph-intra * 2;
+      padding: 0 $cc-button-size + $sph-intra--condensed 0 $sph-intra * 1.5;
       width: 100%;
     }
 
+    &::before {
```

Comment:
I think the same can be achieved with:
``` scss
color: $color-mid;
content: '$';
left: $sph-intra--condensed;
position: absolute;
top: $spv-intra--condensed;
```
Also, do we want the `$` to have Ubuntu Mono font @kwm14?

Reply:
@Caleb-Ellis ooooooh that would be good 💯  can we update @deadlight 😃